plugins/websocket_server/index.html

```diff
@@ -10,8 +10,12 @@
 
   <script>
     // Create the Connection to the Ignition websocket server
+    // This configuration uses SSL and authentication. You can disable both
+    // by replacing `wss://` with `ws://` and remove the `key` parameter.
+    // Make sure to also disable SSL and authentication in the
+    // websocket_server plugin.
     var ign = new Ignition({
-      url: 'ws://localhost:9002',
+      url: 'wss://localhost:9002',
       key: "auth_key"
```

Comment:
Is this the auth key the websocket server expects to receive? Maybe this should be noted in the docs.

This repo does not have the `ign.js` code in it (and rightly so). Where can I read its source?

Reply:
It should contain an ign.js file here: https://github.com/ignitionrobotics/ign-launch/blob/feature/websockets_ssl/plugins/websocket_server/ign.js

Reply:
I had put the authorization documentation here: https://github.com/ignitionrobotics/ign-launch/blob/feature/websockets_ssl/plugins/websocket_server/WebsocketServer.hh#L45. I think yout are looking for it in the README.md, which I'll update.

Reply:
Updated readme: 3273662291b6177336f27e665392898840d6b07a